translations/other_rules.tex/pl.po

```diff
@@ -65,6 +67,17 @@ msgid ""
 "\\end{itemize}\n"
 "\n"
 msgstr ""
+"\\hyperlink{Trading Post}{Punkt Handlowy} i inne efekty umożliwiają wymianę Zasobów z grą według poniższej tabeli.\n"
```

Comment:
Ma ktoś pomysł jak to lepiej ubrać w słowa, żeby wyrazić dobitnie, że handel nie jest możliwy z innymi graczami? wersja en też jakoś mi nie pasuje.. ale może inaczej się nie da?

Reply:
Musiał byś dodać kolejne zdanie, nie bardzo idzie to przełożyć. Niby jest napisane że wymieniasz z grą...

Reply:
a może po prostu wprowadzić słowo "Bank" jako zbiór wszystkich surowców...

Reply:
Bank to spoko pomysł, ale sama gra nie posiada takiego pojęcia, nie chciałbym żeby to było mylące i gracze zastanawiać się jak działa taki Bank. może lepiej to po prostu zostawić w takim razie 🤔

Reply:
Można by na początku po pierwszym zastosowaniu określenia "pula wszystkich surowców" lub "zbiór wszystkich surowców" lub "Gracz dobiera z ogólnej puli surowców..." dopisać w nawiasie (bank) i można by później wykorzystywać, imho...

Reply:
dobry pomysł, zmieniłem :)